code/modules/client/asset_cache.dm

```diff
@@ -479,7 +479,7 @@ GLOBAL_LIST_EMPTY(asset_datums)
 		"stamp-clown" = 'icons/stamp_icons/large_stamp-clown.png',
 		"stamp-deny" = 'icons/stamp_icons/large_stamp-deny.png',
 		"stamp-ok" = 'icons/stamp_icons/large_stamp-ok.png',
-		"stamp-hop" = 'icons/stamp_icons/large_stamp-hop.png',
+		"stamp-xo" = 'icons/stamp_icons/large_stamp-xo.png', //Nsv13 - HoP to XO

```

Comment:
Did you make sure to rename this file as well?

Reply:
Altered now.